.github/actions/zenodo-upload/zenodo_upload.py

```
@@ -0,0 +1,101 @@
+                "creators": [{"name": "Vacanza Team"}]
+            }
+        }
+
+    # Check for existing deposition
+    deposition_id = get_existing_deposition(base_url, token, concept_id)
+
+    # Create new deposition or new version
+    if deposition_id and concept_id:
+        response = requests.post(f"{base_url}/{deposition_id}/actions/newversion", headers=headers)
+        if response.status_code != 201:
+            print(f"Error creating new version: {response.text}")
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)